src/pages/Run/index.tsx

```diff
@@ -84,6 +85,18 @@ const RunPage: React.FC = () => {
           <Box label="Associated assemblies">
             <AssociatedAssemblies rootEndpoint="runs" />
           </Box>
+          <Box label="Additional analyses">
+            <p>
+              Additional annotations produced by workflows run outside the scope
+              of MGnify’s versioned pipelines.
+            </p>
+            <ExtraAnnotations
+              annotationsPageProp="run-annotations-page"
+              annotationsPageSizeProp="run-annotations-page_size"
+              namespace="runs"
+              entityName="run"
```

Comment:
I *think* the page and page-size prop should be inferable from the `namespace`, and therefore maybe don't need to be passed? We have a convention around the app that the page query param would be like `{namespace}page` etc, so often set a namespace to be e.g. `run-annotations-`. See for example the `AssociatedRuns` component in `components/Sample/Run`. I might have missed something though!

Reply:
@SandyRogers I've changed this component to only accept the namespace as a prop. I added a method inside it, that returns the singular version of the namespace, which is used in setting the page and page size values.